lib/chef/knife/windows_listener_create.rb

```diff
@@ -60,6 +60,10 @@ def get_cert_passphrase
         passphrase.strip
       end
 
+      def exitstatus
+        $?
```

Comment:
This should return `$?.exitstatus`, or our check of `if exitstatus` below will eval truthy when it shouldn't.

Reply:
ah yes, that's what i meant to do.  also why i don't particularly like unit tests.